AFQ/api/group.py

```diff
@@ -776,6 +776,48 @@ def upload_to_s3(self, s3fs, remote_path):
         if op.exists(self.afqb_path):
             s3fs.put(self.afqb_path, remote_path, recursive=True)
 
+    def export_group_density(self, boolify=True):
+        """
+        Generate a group density map by combining single subject density maps.
+
+        Parameters
+        ----------
+        boolify : bool
```

Comment:
Sorry - one more thing: with `boolify` set to true, this will count the number of subjects who have at least one streamline in that location (in the mapped template space). Is that really the intended behavior? I am actually not sure what the "correct" behavior should be. That is, how we should treat voxels that have large discrepancies between subjects.